### PR TITLE
errorhandling and whitespace fixes

### LINKED ISSUE #7: ~/.oc/profiles not created automatically
When starting up the cluster for the first time and doing `./oc-cluster list   `  i get an error message saying `ls: /Users/bjartek/.oc/profiles: No such file or directory                                                  

Possible solution, create a better error message saying that you should oc cluster up first?

### PR DESCRIPTION
**Pull request by bjartek**
This started out as a simple PR to just fix the error in the list method, but then i started fixing some more stuff. Hope you like it :)

All the formating is not  2 spaces, not sure what you liked and the original was mixed so i just chose something. Can always change it later if you prefer something else

### CODE REVIEW

**Comment by bjartek** [comment]
Looks like my formatting here is way of, sorry for that.

What settings do you prefer for this? tabs/spaces etc.

**Comment by bjartek** [comment]
This started out as a simple PR to just fix the error in the list method, but then i started fixing some more stuff. Hope you like it :)

File: oc-cluster [hunk 1]
```diff
@@ -5,29 +5,34 @@ OC_CLUSTER_ROUTING_SUFFIX=${OC_CLUSTER_ROUTING_SUFFIX}
...
      echo "There is a cluster already running. You can not run 2 cluster at the same time"
      exit 1
-  fi  
+  fi
 
   if [ "$_profile" == "" ] || [[ $_profile == -* ]]
   then
      echo "Using default profile"
-     _profile="default"
+
```
Comment (jorgemoralespou) [inline oc-cluster]: Why are you removing the default profile?

File: oc-cluster [hunk 2]
```diff
@@ -153,10 +158,16 @@ function destroy {
 }
 
 function list {
+  if [ ! -d "$OPENSHIFT_HOME_DIR/profiles/" ]
```
Comment (jorgemoralespou) [inline oc-cluster]: I would rather show an empty list of profiles

Comment (bjartek) [inline oc-cluster]: Fixed

**Comment by bjartek** [comment]
Let me know if you want me to clean up the history before you merge. I can always do that and force push to this branch

**Pull request merged**